src/renderer/components/+helm-releases/releases.injectable.ts

```diff
@@ -7,17 +7,21 @@ import { asyncComputed } from "@ogre-tools/injectable-react";
 import namespaceStoreInjectable from "../+namespaces/store.injectable";
 import { listReleases } from "../../../common/k8s-api/endpoints/helm-releases.api";
 import clusterFrameContextInjectable from "../../cluster-frame-context/cluster-frame-context.injectable";
+import releaseSecretsInjectable from "./release-secrets.injectable";
 
 const releasesInjectable = getInjectable({
   id: "releases",
 
   instantiate: (di) => {
     const clusterContext = di.inject(clusterFrameContextInjectable);
     const namespaceStore = di.inject(namespaceStoreInjectable);
+    const releaseSecrets = di.inject(releaseSecretsInjectable);
 
     return asyncComputed(async () => {
       const contextNamespaces = namespaceStore.contextNamespaces || [];
 
+      void releaseSecrets.get();
```

Comment:
Why is this enough?

Reply:
`asyncComputed` works as `computed` does. So if you observe some observable, and the value changes, it will re-compute. Here `releaseSecrets` changes when something is installed, so that causes `asyncComputed` to recompute.